osf/migrations/0209_update_egap_schema2.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import unicode_literals
+
+from django.db import migrations
+from osf.utils.migrations import UpdateRegistrationSchemasAndSchemaBlocks
+
+def noop(*args, **kwargs):
+    pass
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('osf', '0208_update_EGAP_schema'),
+    ]
+
+    operations = [
```

Comment:
Let's take out this migration. We should have the management command for this soon, and the typo is minor enough that it can wait until the migration is available.